misc/keygen/Cargo.toml

```
@@ -0,0 +1,18 @@
+[package]
+name = "keygen"
+version = "0.1.0"
+edition = "2021"
+authors = ["demfabris <demfabris@gmail.com>"]
+license = "MIT"
+repository = "https://github.com/libp2p/rust-libp2p"
+keywords = ["peer-to-peer", "libp2p", "networking"]
+categories = ["network-programming", "asynchronous"]
+
+[dependencies]
+structopt = "0.3.26"
+zeroize = "1.5.2"
```

A specific reason to not allow any versions below?
```suggestion
zeroize = "1"
```